src/pages/certificates/CertificateAddForm.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC, useState } from "react";
+import { Button, Form, Textarea } from "@canonical/react-components";
+import { addCertificate } from "api/certificates";
+import { useNotify } from "context/notify";
+
+const CertificateAddForm: FC = () => {
+  const notify = useNotify();
+  const [token, setToken] = useState("");
+
+  const useToken = () => {
+    addCertificate(token)
```

Comment:
I would suggest to add some input sanitisation here, because I noticed that the CLI output includes a "Client lxd-ui certificate add token:" line before the token itself, and it's very easy to carelessly copy-paste the whole thing in this textarea and get a "not authorized" error that really doesn't help much to understand what went wrong. I know I'm being the naughty user here, but prevention is better than cure, right? :slightly_smiling_face: 
```suggestion
    const sanitisedToken = token.trim().split(/\r?\n|\r|\n/g).at(-1) ?? "";
    addCertificate(sanitisedToken)
```

Reply:
yeah, good idea!